enterpriseknowledgegraph/entity_reconciliation/create_entity_reconciliation_job_sample.py

```diff
@@ -0,0 +1,85 @@
+# pylint: disable=too-many-arguments
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# [START enterpriseknowledgegraph_create_entity_reconciliation_job]
+
+from google.cloud import enterpriseknowledgegraph as ekg
+
+# TODO(developer): Uncomment these variables before running the sample.
+# project_id = 'YOUR_PROJECT_ID'
+# location = 'YOUR_GRAPH_LOCATION'                  # Values: 'global'
+# input_dataset = 'YOUR_INPUT_DATASET'              # BigQuery Dataset Name
+# input_table = 'YOUR_INPUT_TABLE'                  # BigQuery Table Name
+# mapping_file_uri = 'YOUR_MAPPING_FILE             # GCS Path. Example: gs://ekg-test-gcs/mapping.yml
+# entity_type = ekg.InputConfig.EntityType.Person   # Refer to https://cloud.google.com/enterprise-knowledge-graph/docs/schema
```

Comment:
While this visually looks fine on GitHub, this may not render well on c.g.c. as embedded samples. Consider moving these to different format, so line length < 80 characters requirement is respected.